roles/deploy/handlers/main.yml

```diff
@@ -57,3 +57,10 @@
   command: bash -lc "bundle exec whenever --set 'environment={{ rails_env }}' --update-crontab"
   args:
     chdir: "{{ current_path }}"
+
+- name: notify slack
+  slack:
+    token: "T02G54U79/BF25P9F7A/DJdtYaLLUpRJPiu72d8NqgGg"
+    msg: '{{ inventory_hostname }} deployed'
```

Comment:
Can we grab the branch/tag/pr that's being deployed whilst we're doing this, and add it to the Slack comment?

Reply:
Sure! let me try